sphinxgallery/docs_resolv.py

```diff
@@ -409,6 +409,8 @@ def embed_code_links(app, exception):
     """Embed hyperlinks to documentation into example code"""
     if exception is not None:
         return
+    if not app.builder.config.plot_gallery:
+        return  # no need to embed
```

Comment:
And also because when writing narrative doc you don't want to plot and wait for all the embedding 


> On 4 août 2015, at 09:35, Loïc Estève <notifications@github.com> wrote:
> 
> In sphinxgallery/docs_resolv.py:
> 
> > @@ -409,6 +409,8 @@ def embed_code_links(app, exception):
> >      """Embed hyperlinks to documentation into example code"""
> >      if exception is not None:
> >          return
> > +    if not app.builder.config.plot_gallery:
> > +        return  # no need to embed
> I missed that in my previous reviews. Why don't you want to put hyperlinks inside the example in case plot_gallery is False? Is it because it can fail if there is a parsing error of the script ?
> 
> —
> Reply to this email directly or view it on GitHub.
>